src/components/quiz-questions/EquilibriumQuestion.tsx

```diff
@@ -1,16 +1,25 @@
-import React, { useContext, useState } from "react";
+import React, { useContext, useRef, useState } from "react";
 import QuizForm from "./QuizForm";
 import VisibilityControl from "../shared/VisibilityControl";
 import { FormState } from "./types";
 import RadioComponent from "../shared/Radio";
-import { PROMPT_TO_ADJUST_B } from "../../constants";
+import { EQUILIBRIUM_QUIZ_ID } from "../../constants";
 import { SimulariumContext } from "../../simulation/context";
+import { Module } from "../../types";
 
 const EquilibriumQuestion: React.FC = () => {
-    const { page } = useContext(SimulariumContext);
+    const id = EQUILIBRIUM_QUIZ_ID;
+    const { page, quizQuestion, module } = useContext(SimulariumContext);
     const [selectedAnswer, setSelectedAnswer] = useState("");
     const [formState, setFormState] = useState(FormState.Clear);
-
+    const DEFAULT_BIG_NUMBER = Infinity;
```

Comment:
Nit: Move outside of class since it's an immutable constant and/or rename to INITIAL_PAGE_NUMBER?

Reply:
Megan: add comment